packages/shared/src/components/sidebar/sections/MainSection.tsx

```diff
@@ -25,6 +37,19 @@ export const MainSection = ({
         }
       : undefined;
 
+    const plus =
+      showPlusSubscription || !isPlus
+        ? {
+            title: 'Upgrade to Plus',
+            path: '/plus',
+            action: onPlusClick,
+            requiresLogin: true,
+            icon: <DevPlusIcon />,
+            color:
+              'text-action-plus-default bg-action-plus-float hover:bg-action-plus-hover active:bg-action-plus-active',
+          }
+        : undefined;
+
```

Comment:
I can still see this when I'm logged in and plus

Reply:
```suggestion
    const plus =
      !isPlus
        ? {
            title: 'Upgrade to Plus',
            path: '/plus',
            action: onPlusClick,
            requiresLogin: true,
            icon: <DevPlusIcon />,
            color:
              'text-action-plus-default bg-action-plus-float hover:bg-action-plus-hover active:bg-action-plus-active',
          }
        : undefined;

```

This should be enough